src/test/java/com/practicesoftwaretesting/APItests/BaseTest.java

```diff
@@ -1,4 +1,4 @@
-package com.practicesoftwaretesting;
+package com.practicesoftwaretesting.APItests;
```

Comment:
```suggestion
package com.practicesoftwaretesting.APItests;
```
Don't use uppercase in packages